raft.go

```diff
@@ -901,7 +901,8 @@ func (r *raft) becomeLeader() {
 	// uncommitted log quota. This is because we want to preserve the
 	// behavior of allowing one entry larger than quota if the current
 	// usage is zero.
-	r.reduceUncommittedSize(payloadSize(emptyEnt))
```

Comment:
> // As a special case ...

Now that this is a no-op (it wasn't always), this isn't exactly a special case. I think we can remove the original comment and the commented-out line of code and say something simpler like:
> The payloadSize of an empty entry is 0 (see TestPayloadSizeOfEmptyEntry), so the preceding log append does not count against the uncommitted log quota of the new leader. In other words, after the call to appendEntry, r. uncommittedSize is still 0.